assets/css/admin.css

```diff
@@ -147,4 +147,26 @@
 	left: 2px;
 	width: 23px;
 	height: 37px;
-}
\ No newline at end of file
+}
+
+#patreon_setup_screen {

```

Comment:
I think this code is also in #79 -- is that intentional?

Reply:
Removed.